src/Symfony/Component/Security/Core/Tests/Authentication/Token/AbstractTokenTest.php

```diff
@@ -212,53 +209,59 @@ public function testSetUserSetsAuthenticatedToFalseWhenUserChanges($firstUser, $
     }
 
     public function getUserChanges()
+    {
+        $user = $this->getMockBuilder('Symfony\Component\Security\Core\User\UserInterface')->getMock();
+
+        return array(
+            array('foo', 'bar'),
+            array('foo', new TestUser('bar')),
+            array('foo', $user),
+            array($user, 'foo'),
+            array($user, new TestUser('foo')),
+            array(new TestUser('foo'), new TestUser('bar')),
+            array(new TestUser('foo'), 'bar'),
+            array(new TestUser('foo'), $user),
+        );
+    }
+
+    /**
+     * @group legacy
+     *
+     * @dataProvider getUserChangesAdvancedUser
+     */
+    public function testSetUserSetsAuthenticatedToFalseWhenUserChangesdvancedUser($firstUser, $secondUser)
```

Comment:
There's an 'A' missing here to complete the word 'Advanced' :)

Reply:
Heh, you're completely right! If you wish, you could make a PR to fix this (you can just click the edit file). Gives you a nice contributor tag

Reply:
Will do =) Thank you.